CriticalMass/BikeAnnotationView.swift

```diff
@@ -7,7 +7,15 @@
 
 import MapKit
 
-class BikeAnnoationView: MKAnnotationView {
+protocol AnnotationView: MKAnnotationView {
+    associatedtype T = Any
```

Comment:
Do you need to pin Any to an associatedType and then in an implementation say `T = Friend`?

Reply:
The Any is a default value in case the AnnotationView doesn't need an Object.  Swift currently doesn't support optional Associated type so this is a workaround for that.